scripts/setup

```diff
@@ -215,7 +226,8 @@ info_msg "- Circulation policies: ${CREATE_LAZY} ${DONT_STOP}"
 eval ${PREFIX} pipenv run invenio fixtures create --pid_type cipo ${DATA_PATH}/circulation_policies.json --append ${CREATE_LAZY} ${DONT_STOP}
 info_msg "Indexing Circulation policies:"
 eval ${PREFIX} pipenv run invenio utils reindex -t cipo --yes-i-know
-eval ${PREFIX} pipenv run invenio utils runindex --raise-on-error
+info_msg "Run indexing concurrency=${CONCURRENCY}:"
+eval ${PREFIX} pipenv run invenio utils runindex --concurrency ${CONCURRENCY} --raise-on-error
```

Comment:
Is `--concurrency` works without `-d` or `--delayed` option?
see https://github.com/rero/rero-ils/blob/dev/rero_ils/modules/cli.py#L902

According to me, you must use `--delayed`